### DISCUSSION TITLE
Faster way to add categories to entry

### DISCUSSION THREAD

**Discussion by nitech** [craftcms/cms#8950] [Ideas] [answered]
### Description
I would like a faster way to add categories to entries. 

Right now, I have to click a button, and then traverse the tree or list of categories.

I would like to be able to add categories by either: 

a. Typing and having a list (with full tree if category is a structure) populated below. Key navigation, Enter to select.
b. Selecting by navigating tree (same as today).

**Comment by nitech** [comment]
Perhaps use [Selectize](https://selectize.github.io/selectize.js/)?

**Comment by bencroker** [comment, marked as answer]
This discussion can probably be closed, as in Craft 4, it is possible to search and filter when selecting related elements (including categories).

<img width="1013" alt="Screenshot 2023-01-12 at 20 36 52" src="https://user-images.githubusercontent.com/57572400/212224401-c19b2d19-eaaa-460f-81c7-025c034bf6db.png">
